__tests__/InlineHtmlStripStylesTransformer.test.ts

```diff
@@ -0,0 +1,214 @@
+/*
+ * Code is inspired by
+ * https://github.com/kulshekhar/ts-jest/blob/25e1c63dd3797793b0f46fa52fdee580b46f66ae/src/transformers/hoist-jest.spec.ts
+ * original license:
```

Comment:
We already include this license within our package because we use `ts-jest` as our dependency. Please remove it here and in `InlineHtmlStripStylesTransformer.ts`. The "code inspiration" notice is good to stay of course.

Reply:
:+1: will update